src/TypeSpec.Extension/Emitter.Csharp/src/lib/serviceAuthentication.ts

```diff
@@ -28,6 +30,22 @@ export function processServiceAuthentication(
                         }
                     }
                     break;
+                case "http":
+                    const schemeOrApiKeyPrefix = scheme.scheme;
+                    if (
+                        schemeOrApiKeyPrefix === "basic" ||
+                        schemeOrApiKeyPrefix === "bearer"
+                    ) {
+                        logger.warn(
+                            `{scheme.scheme} auth method is currently not supported.`
```

Comment:
Shall we use 'schemeOrApiKeyPrefix' here to be consistent?

Reply:
updated